steps/browserstack-bitrise-espresso-step/1.0.0/step.yml

```diff
@@ -0,0 +1,256 @@
+- devices_list: null
+  opts:
+    category: Devices
+    description: "Name of one or more device-OS combination in new line \nFor example:\n\n`Samsung
+      Galaxy S9 Plus-9.0`\n\n`Google Pixel 3a-9.0`\n"
+    is_expand: true
+    is_required: true
+    summary: |
+      Name of one or more device-OS combination in new line from the [device list](https://www.browserstack.com/list-of-browsers-and-platforms/app_automate) . For example:
+
+      `Samsung Galaxy S9 Plus-9.0`
+
+      `Google Pixel 3a-9.0`
+    title: Devices
+- instrumentation_logs: true
+  opts:
+    category: Debug logs
+    description: Instrumentation logs are a comprehensive record of your Espresso
+      test executions that helps you identify all the steps executed in the test.
+    summary: Enable Instrumentation Logs generated by AndroidJunitRunner
+    title: Instrumentation logs
+    value_options:
+    - true
+    - false
+- network_logs: false
+  opts:
+    category: Debug logs
+    description: 'Network logs captures the network traffic, latency, HTTP requests
+      and responses in the HAR (HTTP Archive) format. '
+    summary: Enable network traffic, latency, HTTP requests and responses in the HAR
+      (HTTP Archive) format
+    title: Network Logs
+    value_options:
+    - true
+    - false
+- device_logs: false
+  opts:
+    category: Debug logs
+    description: Device logs are system logs specific to your application generated
+      by Android logcat that help you debug any test crashes.
+    summary: Enable logs generated by Android logcat
+    title: Android logcat logs
+    value_options:
+    - true
+    - false
+- debug_screenshots: false
+  opts:
+    category: Debug logs
+    description: Screenshots makes it easier to identify the exact step in your test
+      where the failure occurred, especially any layout or design-related issues.
+    summary: Capture the screenshots in your tests
+    title: Capture Screenshots
+    value_options:
+    - true
+    - false
+- opts:
+    category: Debug logs
+    description: Video recording helps you view the entire test execution and helps
+      debug any failed test.
+    summary: Enable the video of the test run
+    title: Video recording
+    value_options:
+    - true
+    - false
+  video_recording: true
+- opts:
+    category: Test Configuration
+    description: A project name helps you group multiple Espresso test executions
+      under a single project.
+    summary: Project name of the tests.
+    title: Project name
+  project: null
+- filter_test: null
+  opts:
+    category: Test Configuration
+    description: |
+      Provide key-value pairs of filters to run tests: class, package, annotation, size
+
+      For example: class com.foo.AddToCartClass,class com.foo.CheckoutClass
+
+      package com.foo
+
+      Learn more.(https://www.browserstack.com/docs/app-automate/espresso/select-test-cases)
+    summary: "Key-value pairs of filters to run tests from supported test filtering
```

Comment:
This summary is a bit too long and would look broken on the UI.

Reply:
"Provide a comma-separated list of class or test names followed by supported [filtering strategies](https://www.browserstack.com/docs/app-automate/espresso/select-test-cases)."